source/utilities.tex

```diff
@@ -16312,11 +16312,11 @@
 \end{codeblock}
 
 \begin{note} This requirement gives well defined results for reference types, void
-types, array types, and function types.\end{note} Access checking is performed as
-if in a context unrelated to \tcode{To} and \tcode{From}. Only the validity of
-the immediate context of the expression of the \grammarterm{return-statement}
-(including conversions to the return type) is considered. \begin{note} The
-evaluation of the conversion can result in side effects such as the
+types, array types, and function types.\end{note} Access checking is performed
+in a context unrelated to \tcode{To} and \tcode{From}. Only the validity of
+the immediate context of the \grammarterm{expression} of the \tcode{return} statement
+(including conversions to the returned object or reference) is considered. \begin{note} The
```

Comment:
Did you mean to retain "conversion to" here? @zygoloid suggested "initialization of" instead. It seems more accurate to "initialize a reference" than to "convert to a reference".